src/d3d12/nvapi_d3d12_device.cpp

```diff
@@ -105,17 +107,36 @@ namespace dxvk {
         return deviceExt;
     }
 
-    Com<ID3D12GraphicsCommandListExt> NvapiD3d12Device::GetCommandListExt(ID3D12GraphicsCommandList* commandList) {
+    d3d12_graphics_command_list_ext_iface* NvapiD3d12Device::GetCommandListExt(ID3D12GraphicsCommandList* commandList, uint32_t* interfaceVersion) {
```

Comment:
I'm a bit conflicted about what would be less awkward here, this or returning `std::optional<std::pair<d3d12_graphics_command_list_ext_iface, uint32_t>>` containing both the interface and its version. I'll leave this one to Jens.

Reply:
I varied from your suggestion a bit and decided to separate out building the `NvapiD3d12Device::CommandListExtWithVersion` from the emplace, to improve debuggability.